source/code-generation/query-converter/convert-triggers.txt

```diff
@@ -63,9 +63,11 @@ Steps
       #. Click the :icon-fa5:`chevron-right` icon next to
          :guilabel:`Triggers`.
 
-         You can toggle triggers for conversion by clicking
-         the :icon-fa5:`check-square` icon next to a trigger's
-         name. All triggers are selected by default.
+         .. tip:
+
+            Click the :icon-fa5:`check-square` icon next to a trigger's 
+            name to toggle it for conversion. All triggers 
+            are selected by default.
```

Comment:
Same restructure as above. (Also same minor question.) 

General format s/b: To accomplish AB, do CD. 
So, in this case:
To toggle a trigger for conversion, click the check-square ...